src/ansys/fluent/core/launcher/fluent_container.py

```diff
@@ -228,6 +228,9 @@ def configure_container_dict(
         host_mount_path = volumes_string.replace(":" + container_mount_path, "")
         logger.debug(f"host_mount_path: {host_mount_path}")
         logger.debug(f"container_mount_path: {container_mount_path}")
+    logger.warning(
+        f"Starting Fluent container mounted to {host_mount_path}, with this path available as {container_mount_path} for the Fluent session running inside the container."
+    )
```

Comment:
Although I suggested this string, I think we should still briefly review this string with the rest of the team, to make sure it is clear and understandable enough to everyone. The wording can probably still be improved.

Also, @PipKat I believe your input would be appreciated here, if you can find the time.